### PR TITLE
Richard grants round design issue

### LINKED ISSUE #209: GrantsRound Design Issue
Bildschirmfoto 2021-09-11 um 16 44 02

- use the codebase of `GrantDetailsRow.vue` for `GrantRoundDetailsRow.vue`
to ensure same style .

reference screenshot of current` GrantDetailsRow.vue` of  difference : 

Bildschirmfoto 2021-09-11 um 16 55 01

- add funds button : remove green from icon
- share will be "tweet" + tweet icon ( from fusion )
- description :  each <p> should be `<p class="text-intend"> `

### PR DESCRIPTION
**Pull request by scco**
closes #209

- fixed the .intent in index.css  { was somehow not working for me ...  }
- fixing positioning of GrantRoundDetailsRow text elements / button
- removing the green from Add Funds
- add Tweet Button Design
- generating a rudimentary but probably not working tweet-link ( i created an issue #212 ) for this

### CODE REVIEW

**Review by mds1 — APPROVED** [review]

File: app/src/views/GrantRound.vue [hunk 1]
```diff
@@ -75,25 +75,37 @@
...
     </div>
 
     <!-- Description -->
     <SectionHeader title="Description" />
+
     <div class="border-b border-grey-100">
-      <p style="white-space: pre-line" class="intent px-4 md:px-12 py-24 mx-auto max-w-6xl">
-        {{ grantRoundMetadata?.description }}
-      </p>
+      <div class="px-4 md:px-12 py-12 mx-auto max-w-6xl">
```
Comment (thelostone-mc) [inline app/src/views/GrantRound.vue:104]: This does mean everything would be not aligned right. 
I'd say keep white-space: preline until you've thrown in a CRLF / ENTER 

Right now line breaks as stored as `/n` and we don't want to store the entire HTML of description onto IPFS

File: app/src/index.css [hunk 2]
```diff
@@ -56,7 +56,7 @@
 
 /* Add styles to components */
 @layer components {
-  .indent {
+  .intent {
```
Comment (thelostone-mc) [inline app/src/index.css:59]: this seems like a really generic name ! 
could we do a different name ?

File: app/src/views/GrantRound.vue [hunk 3]
```diff
@@ -174,7 +186,7 @@ import {
 import { GrantRound, GrantRoundMetadata, Breadcrumb } from '@dgrants/types';
 
 // --- Icons ---
-import { ArrowToprightIcon as ShareIcon } from '@fusion-icons/vue/interface';
+import { TwitterIcon as TweetIcon } from '@fusion-icons/vue/interface';
```
Comment (thelostone-mc) [inline app/src/views/GrantRound.vue:189]: ^ do we need the rename ? why not use TwitterIcon as is

File: app/src/views/GrantRound.vue [hunk 4]
```diff
@@ -75,25 +75,37 @@
...
+        <!-- Tweet -->
+        <a
+          target="_blank"
+          :href="
+            'https://twitter.com/intent/tweet?text=' +
+            grantRoundMetadata?.name +
+            '&url=https://grants.gtcdao.net' +
+            $route.path
+          "
+          class="flex items-center gap-x-2 cursor-pointer group ml-auto"
```
Comment (thelostone-mc) [inline app/src/views/GrantRound.vue:87]: Also add no `noreferrer noopener`

**Review by thelostone-mc — CHANGES_REQUESTED** [review]
Left a few comments

File: app/src/views/GrantRound.vue (hunk 1 above)
Comment (scco) [reply app/src/views/GrantRound.vue:104]: changed to "whitespace-pre-line"  but later maybe revisit to have it as i want in design.

File: app/src/index.css (hunk 2 above)
Comment (scco) [reply app/src/index.css:59]: i think i mixed up also   .intent with .indent ...  so "indent" or "indentation" is the right world in english.
but as we wana use this just for <p> with long TEXT  i renamed it 

.text-indent

File: app/src/views/GrantRound.vue (hunk 3 above)
Comment (scco) [reply app/src/views/GrantRound.vue:189]: im ok with keep it TwitterIcon. question of taste.

File: app/src/views/GrantRound.vue (hunk 4 above)
Comment (scco) [reply app/src/views/GrantRound.vue:87]: ok

**Comment by scco** [comment]
added another fix to have links in the links section be underlined.

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**